stubs/tensorflow/tensorflow/keras/callbacks.pyi

```diff
@@ -0,0 +1,188 @@
+from _typeshed import Incomplete
+from collections.abc import Callable, Iterable, Mapping, Sequence
+from typing import Any, Literal
+from typing_extensions import TypeAlias
+
+import tensorflow as tf
+from tensorflow.keras import Model
+from tensorflow.keras.optimizers.schedules import LearningRateSchedule
+
+_Logs: TypeAlias = Mapping[str, Any] | None | Any
+
+class Callback:
+    model: Model  # Model[Any, object]
+    params: dict[str, Any]
+    # def set_model(self, model: Model[Any, object]) -> None: ...
+    def set_model(self, model: Model) -> None: ...
+    def set_params(self, params: dict[str, Any]) -> None: ...
+    def on_batch_begin(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_batch_end(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_epoch_begin(self, epoch: int, logs: _Logs = None) -> None: ...
+    def on_epoch_end(self, epoch: int, logs: _Logs = None) -> None: ...
+    def on_predict_batch_begin(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_predict_batch_end(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_predict_begin(self, logs: _Logs = None) -> None: ...
+    def on_predict_end(self, logs: _Logs = None) -> None: ...
+    def on_test_batch_begin(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_test_batch_end(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_test_begin(self, logs: _Logs = None) -> None: ...
+    def on_test_end(self, logs: _Logs = None) -> None: ...
+    def on_train_batch_begin(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_train_batch_end(self, batch: int, logs: _Logs = None) -> None: ...
+    def on_train_begin(self, logs: _Logs = None) -> None: ...
+    def on_train_end(self, logs: _Logs = None) -> None: ...
+
+# A CallbackList has exact same api as a callback,
+# but does not actually subclass it.
+class CallbackList:
+    def __init__(
+        self,
+        callbacks: Sequence[Callback] | None = None,
+        add_history: bool = False,
+        add_progbar: bool = False,
+        # model: Model[Any, object] | None = None,
+        model: Model | None = None,
+        **params: Any,
+    ) -> None: ...
+    # def set_model(self, model: Model[Any, object]) -> None: ...
```

Comment:
I'd drop commented out stub and when Model is added later as generic ci checks should notice.

Reply:
Done in 3b965af2a